train_configs/debug_model.toml

```diff
@@ -37,7 +37,7 @@ steps = 10
 data_parallel_degree = -1
 tensor_parallel_degree = 1
 pipeline_parallel_degree = 1
-fp8_linear = ""
+fp8_linear = '' # ['', 'dynamic'] See torchtitan/float8_linear.py for more detail
```

Comment:
IMO we should put details about a config into the helper message in `config_manager.py`. Here we can simply have `# ['', 'dynamic']` as the possible options, similar to the `norm_type` config. But I'm ok with also putting them here.

Reply:
yeah that makes sense, I was trying to find the best place for this one I can update this PR